tests/unit/amo/components/TestCollection.js

```diff
@@ -1268,6 +1268,66 @@ describe(__filename, () => {
         username: detail.author.username,
       }),
     );
+
+    sinon.assert.notCalled(root.instance().props.history.push);
+  });
+
+  it('updates the page when removeAddon removes the last addon on the page', () => {
+    const history = createFakeHistory();
```

Comment:
You probably want to use it, you should pass it a `location` that is created a few lines after:

``` js
const location = createFakeLocation({ query: { page, collection_sort: sort } });
const history = createFakeHistory({ location });
```

Then you can pass both `location` and `history` and use the latter in `assert.test.calledWith(histor.push, ...)`.